brainiak/factoranalysis/htfa.py

```diff
@@ -64,23 +64,19 @@ class HTFA(TFA):
     Parameters
     ----------
 
-    R : list of 2D arrays, element i has shape=[n_voxel, n_dim]
-        Each element in the list contains the scanner coordinate matrix
-        of fMRI data of one subject.
-
-    K : int, default: 50
+    K : int
         Number of factors to compute.
 
+    n_subj : int, default: 2
```

Comment:
We should remove the default from the docstring too.

As an aside, having to change the docstring now is a good reason to not put the default in the docstring in the first place, given that it is visible in the signature and Sphinx adds it to the HTML docs.